poem/todo-postgres/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "postgres"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+
+[dependencies]
+poem = "1.3.35"
+serde = "1.0"
+shuttle-service = { version = "0.7.2", features = ["web-poem"] }
+shuttle-shared-db = { version = "0.7.2", features = ["postgres"] }
```

All shuttle crates should be bumped to `0.9.0` now, here and in other `Cargo.toml`s.
```suggestion
shuttle-service = { version = "0.9.0", features = ["web-poem"] }
shuttle-shared-db = { version = "0.9.0", features = ["postgres"] }
```